x-pack/platform/plugins/shared/streams/server/lib/streams/component_templates/logs_layer.ts

```diff
@@ -245,7 +248,7 @@ export function addAliasesForNamespacedFields(
   );
 
   // Add aliases defined in the base mappings
-  Object.entries(baseMappings).forEach(([key, fieldDef]) => {
+  Object.entries(baseMappings || {}).forEach(([key, fieldDef]) => {
```

Comment:
If this refer to the `baseMappings` defined above, there might be an issue with the type in [L108](https://github.com/elastic/kibana/pull/228415/files#diff-2e96083589d634ea495c710125097c290496e03cc08e2dfd35749f33c068b442R108) since that's statically defined and should always exist.

Reply:
that's a leftover, removed